plugins/wporg-5ftf/includes/pledge.php

```diff
@@ -117,19 +125,158 @@ function register_custom_post_type() {
  * @return void
  */
 function register_custom_post_status() {
+	$label_count = _n_noop(
+		'Deactivated <span class="count">(%s)</span>',
+		'Deactivated <span class="count">(%s)</span>',
+		'wporg-5ftf'
+	);
 	register_post_status(
-		FiveForTheFuture\PREFIX . '-deactivated',
+		DEACTIVE_STATUS,
 		array(
-			'label'       => __( 'Deactivated', 'wporg-5ftf' ),
-			'label_count' => _n_noop( 'Deactivated <span class="count">(%s)</span>', 'Deactivated <span class="count">(%s)</span>', 'wporg-5ftf' ),
-			'public'      => false,
-			'internal'    => false,
-			'protected'   => true,
-			CPT_ID        => true, // Custom parameter to streamline its use with the Pledge CPT.
```

Comment:
I'm not sure what this was for, but I can re-add it if it's necessary.

Reply:
I added this preemptively to make it easy to retrieve custom statuses that are only for the pledge post type. With that value, you can do this:

```
get_post_stati(
	array(
		CPT_ID => true,
	)
);
```

But unless we add more custom ~stati~ statuses, it's probably fine to remove.